### PR TITLE
shell completion: use more constants in the code

### PR DESCRIPTION
**Pull request by Luap99**
see individual commits

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: libpod/events/config.go [hunk 1]
```diff
@@ -117,8 +119,6 @@ const (
 
 	// Attach ...
 	Attach Status = "attach"
-	// AutoUpdate ...
-	AutoUpdate Status = "auto-update"
```
[the remark this answers is not in the document]
Comment (Luap99) [reply libpod/events/config.go:121]: https://github.com/containers/podman/issues/14283

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
nice work @Luap99

**Pull request merged**